components/sidebar-navigation/Components/LinkItem.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { cnb } from "cnbuilder";
+
+/**
+ * Link Item for Sidebar Navigation
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export default function LinkItem({ level, url, shortName, active }) {
+  return (
+    <a
+      className={cnb(
+        "su-block su-border-l-5 hocus:su-underline hocus:su-text-digital-red hocus:su-border-digital-red dark:hocus:su-text-dark-mode-red dark:hocus:su-border-dark-mode-red",
```

Comment:
@iamrentman I personally like the font size in the preview, but if I compare it to Figma, the font size for all the menu links is 18px on desktop which is quite a bit smaller than the preview. Is this intentional?

Reply:
Intentional only because I preferred this to matching the design as well. :) I think it is easier to read when looking at the actual page rendered in the browser.